america.py

```diff
@@ -8,22 +8,31 @@
 country = "the united states of america"
 
 # 1. print the country in *lower case*
+print(country.lower())
 
 # 2. print country in *title case*
+country = (country.title())
 
 # 3. print country in *upper case*
+print(country.upper())
 
 # 4. print the length of the country string as:
 #    string length of country: <length>
+print(f"string length of country:", (len(country)))
 
 # 5. slice the string "america" from country, upper case the 'a',
 #    and store it in a variable named `short_name` with a value "America"
+short_name = country[21:28].title()
+print(short_name)
 
 # 6. concantentate a string value with `short_name` to
 #    produce the name of the continent where the country resides
 #    and print the result:
 #    <title case country> is located within <title case continent>
+continent = "north america"
+print(f"{short_name} is located within {continent.title()}")
```

Comment:
Output should be "The United States Of America is located within North America" using the title case of the *full* country name.